contrib/storage-hive/core/src/test/java/org/apache/drill/exec/hive/TestHiveStorage.java

```diff
@@ -510,6 +517,38 @@ public void testIgnoreSkipHeaderFooterForSequencefile() throws Exception {
         .go();
   }
 
+  @Test
```

Comment:
I think it's fine to leave the Decimal related issues alone, as long as this PR does not introduce new issue to decimal functions.